README.md

```diff
@@ -6,7 +6,22 @@ Static typechecker for Erlang based on set-theoretic types.
 
 ## User-level documentation
 
-Yet to come...
+* `make` will generate a standalone portable escript called `ety` inside the directory
+  `_build/default/bin`
+* `./ety -h` for help
+* To check a single module file `hello.erl` execute `./ety hello.erl`
+* To check a rebar project `hello.erl`
+    * put `ety` inside the root
```

Comment:
you don't need to do that. You can just `PATH_TO_ETYLIZER/ety`

Reply:
Fixed